extensions/amp-iframe/1.0/test/standalone/test-bento-iframe.js

```diff
@@ -20,10 +22,14 @@ describes.realWin(
 
     async function waitRendered() {
       await element.getApi();
+      await flush();
```

Comment:
Why do we need flush here?

Reply:
Added it because it's in the amp test. 

I remember Jake added them in as part of work to isolate tests from one another by flushing preact hooks tasks, so I think it's important